src/main/java/com/session/service/configuration/AppConfiguration.java

```diff
@@ -0,0 +1,52 @@
+package com.session.service.configuration;
+
+import static com.session.service.logging.SessionsClient.info;
+
+public class AppConfiguration {
```

Comment:
This wont be needed for the library - this sort of thing will be the responsibility of the app using this lib.